src/pheromones/hivemind-host/include/hivemind-host/HiveMindHostDeserializer.h

```diff
@@ -0,0 +1,22 @@
+#ifndef __HIVEMINDHOSTDESERIALIZER_H_
```

Comment:
Document?
I don't know if this would belong in here or in the Interface (or elsewhere?), but it would be handy to explain briefly the usage of the `HiveMindHostSerializer/Deserializer` classes. Like saying that these are the "entrypoint" to be used to send and recieve some data between the HB and the host. And also explaining that the "serialize" method also _sends_ the data over the stream. For me, "serializing" and "sending" are two different concepts, and the fact that they are merged was a bit confusing at first.

Reply:
Just to be clear they are only sending them to the IProtobufStream, theyr are not sent to the wire, thats the stream job send them (or he could accumulate them, or send them to another process/thread, whatever)